Makefile.libcompat

```diff
@@ -108,7 +108,7 @@ build${libcompat}: .PHONY
 	${_+_}cd ${.CURDIR}; \
 	    ${LIBCOMPATWMAKE} -f Makefile.inc1 -DNO_FSCHG libraries
 .if ${MK_COMPAT_CHERIABI} == "yes" && ${MK_CHERI_PURE} == "yes"
-# To build a purecap world against /usr/libcheri, we need internal libs.
+# To build a purecap world against /usr/lib64c, we need internal libs.
 # Skip amu because it needs unifdef which isn't available here and we
 # don't build amd (and it's going away).
```

Comment:
Not related to this change, but the amu bit is obsolete since we've merged the amd removal.